### PR TITLE
Bta 567: How to pay corporation tax

### PR DESCRIPTION
**Pull request by AndrewShanks**

### CODE REVIEW

File: app/views/ct/how_to_pay_corporation_tax.scala.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        <h2>@messages("ct.how_to_pay.deadlines.heading")</h2>
+
+        <p>@messages("ct.how_to_pay.deadlines.body")</p>
+
+        <h2>@messages("ct.how_to_pay.up_to_1point5_million.heading")</h2>
+
+        <p>
+            @messages("ct.how_to_pay.up_to_1point5_million.body")
+            <a id = "year-company-set-up" href = "@appConfig.getGovUrl("firstCompanyAccountsAndReturn")"
```
Comment (Fenwick17) [inline app/views/ct/how_to_pay_corporation_tax.scala.html]: Can you remove spaces by the =

File: app/views/ct/how_to_pay_corporation_tax.scala.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+        <p>
+            @messages("ct.how_to_pay.you_must_pay_")
+            <a id = "instalments" href = "@appConfig.getGovUrl("corporationTaxInInstalments")"
+            data-journey-click="HowToPayCt:click:Instalments">@messages("ct.how_to_pay.instalments")</a>.
+        </p>
+
+        <p>
+            @messages("ct.how_to_pay.make_sure_you_pay_")
+            <a id = "charge-you-interest" href = "@appConfig.getGovUrl("corporationTaxInterestCharges")"
```
Comment (Fenwick17) [inline app/views/ct/how_to_pay_corporation_tax.scala.html]: Can you remove spaces by the =

File: app/views/ct/how_to_pay_corporation_tax.scala.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            data-journey-click="HowToPayCt:click:ChargeYouInterest">@messages("ct.how_to_pay.charge_you_interest")</a>
+            @messages("ct.how_to_pay.if_you_do_not_pay_on_time_")
+            <a id = "pay-you-interest" href = "@appConfig.getGovUrl("corporationTaxInterestRefund")"
```
Comment (Fenwick17) [inline app/views/ct/how_to_pay_corporation_tax.scala.html]: Can you remove spaces by the =

File: app/views/ct/how_to_pay_corporation_tax.scala.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+    <h2>@messages("ct.how_to_pay.deadlines.heading")</h2>
+
+    <p>@messages("ct.how_to_pay.deadlines.body")</p>
+
+    <h2>@messages("ct.how_to_pay.up_to_1point5_million.heading")</h2>
+
+    <p>
+      @messages("ct.how_to_pay.up_to_1point5_million.body")
+      <a id="year-company-set-up" href="@appConfig.getGovUrl("firstCompanyAccountsAndReturn")"
```
Comment (Fenwick17) [inline app/views/ct/how_to_pay_corporation_tax.scala.html]: Can you add `target="_blank" rel="external noopener"`

**Comment by chris-ianson** [comment]
approved by qa

**Pull request merged**